src/lib/cimbar_js/cimbar_zstd_js.h

```diff
@@ -0,0 +1,20 @@
+/* This code is subject to the terms of the Mozilla Public License, v.2.0. http://mozilla.org/MPL/2.0/. */
+#ifndef CIMBAR_ZSTD_JS_API_H
+#define CIMBAR_ZSTD_JS_API_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+int cimbarz_init_decompress(unsigned char* buffer, unsigned size);
+int cimbarz_get_filename(char* buffer, unsigned size);
+int cimbarz_get_bufsize();
+int cimbarz_decompress_read(unsigned char* buffer, unsigned size);
```

Comment:
After `cimbard_finish_copy()` succeeds and we have our full zstd bytes reassembled, we call:
`cimbarz_init_decompress()` -> to set up the operation
`cimbarz_decompress_read()` -> until we don't have any more data to decompress

`cimbarz_get_filename` is a stub implementation, not implemented yet.